oaknational/Oak-Web-Application
# ENG-797: Replace Confirmic with oak-consent-client x oak-components

## Description

Music year: [1975](https://www.youtube.com/watch?v=jQAK6sVovUk)

* Replaces the Confirmic consent UI with oak-consent-client and oak-components
* Uses the existing logic to enable and disable services when consent is granted or denied
  * Hopefully this means everything will work as it currently does through Confirmic
* Some refactoring to minimise the API surface area inside OWA 

## How to test

1. Go to https://deploy-preview-2518--oak-web-application.netlify.thenational.academy
2. You should see the consent banner at the bottom of the screen
3. You should be able to grant and decline policies through the banner and settings modal

📝 **You can delete the `cookie-consent` cookie or start a new incognito session to re-trigger the consent banner**

## Screenshots

https://github.com/oaknational/Oak-Web-Application/assets/122096/bf2d953b-72ff-4536-be3f-2e257105ec7f

## Checklist
